repository/i.json

```diff
@@ -1093,6 +1093,18 @@
 				}
 			]
 		},
+		{
+			"name": "Ipynb Preview as Markdown",
+			"details": "https://github.com/valentinbarral/IpynbPreviewAsMarkdown",
+			"labels": ["text manipulation", "formatting"],
+			"releases": [
+				{
+					"platforms": "*",
```

Comment:
If it's `*` you can remove the platforms key.

Reply:
Hi, 
you are right, it makes more sense to open a separate window to show the preview. I have modified the code so that it now behaves that way.
I have also removed the platforms key as you suggested.